ui/admin/app/routes/scopes/scope/auth-methods/auth-method.js

```diff
@@ -7,21 +7,45 @@ import Route from '@ember/routing/route';
 import { action } from '@ember/object';
 import { notifySuccess, notifyError } from 'core/decorators/notify';
 import { inject as service } from '@ember/service';
+import { paramValueFinder } from 'admin/utils/param-value-finder';
+
 export default class ScopesScopeAuthMethodsAuthMethodRoute extends Route {
   // =services
 
   @service store;
+  @service can;
+  @service router;
 
   // =methods
 
   /**
    * Load an auth method by ID.
    * @param {object} params
    * @param {string} params.auth_method_id
-   * @return {authMethodModel}
+   * @return {AuthMethodModel}
    */
   async model({ auth_method_id }) {
```

Comment:
I am aware you did not introduce this change, but do we need this `async` here?

Reply:
we do need the async since we are returning a promise, I realize now this is a bit vague because the comment should specify a promise is returned. I'll update the comments to be more specific!

Reply:
Comment totally for learning purposes: I am aware we return a promise by the usage of `reload: true`, as I [understood from documentation](https://api.emberjs.com/ember-data/release/classes/Store/methods/findRecord?anchor=findRecord), but didn't though we need to declare the method `async` since we are not waiting for the promise.

I will delete all my comments about `async` within `model` methods.